pygments/lexers/arturo.py

```diff
@@ -0,0 +1,312 @@
+                'extend', 'extract', 'factors', 'false', 'fdiv', 'filter',
+                'first', 'flatten', 'floor', 'fold', 'from', 'function',
+                'gamma', 'gcd', 'get', 'goto', 'hash', 'help', 'hypot', 'if',
+                'in', 'inc', 'indent', 'index', 'infinity', 'info', 'input',
+                'insert', 'inspect', 'intersection', 'invert', 'join', 'keys',
+                'kurtosis', 'last', 'let', 'levenshtein', 'lighten', 'list',
+                'ln', 'log', 'loop', 'lower', 'mail', 'map', 'match', 'max',
+                'maybe', 'median', 'min', 'mod', 'module', 'mul', 'nand',
+                'neg', 'new', 'nor', 'normalize', 'not', 'now', 'null', 'open',
+                'or', 'outdent', 'pad', 'panic', 'path', 'pause',
+                'permissions', 'permutate', 'pi', 'pop', 'pow', 'powerset',
+                'powmod', 'prefix', 'print', 'prints', 'process', 'product',
+                'query', 'random', 'range', 'read', 'relative', 'remove',
+                'rename', 'render', 'repeat', 'replace', 'request', 'return',
+                'reverse', 'round', 'sample', 'saturate', 'script', 'sec',
+                'sech', 'select', 'serve', 'set', 'shl', 'shr', 'shuffle',
+                'sin', 'sinh', 'size', 'skewness', 'slice', 'sort', 'split',
+                'sqrt', 'squeeze', 'stack', 'strip', 'sub', 'suffix', 'sum',
+                'switch', 'symbols', 'symlink', 'sys', 'take', 'tan', 'tanh',
+                'terminal', 'to', 'true', 'truncate', 'try', 'type', 'union',
+                'unique', 'unless', 'until', 'unzip', 'upper', 'values', 'var',
+                'variance', 'volume', 'webview', 'while', 'with', 'wordwrap',
+                'write', 'xnor', 'xor', 'zip'
+            ), prefix=r'\b', suffix=r'\b'), Name.Builtin)
+        ],
+            'builtin-predicate-functions': [
+                (
+                    r'\b('
+                    r'all|and|any|ascii|attr|attribute|attributeLabel|binary|block'
+                    r'|char|contains|database|date|dictionary|empty|equal|even'
+                    r'|every|exists|false|floatin|function|greater|'
+                    r'|greaterOrEqual|if|in|inline|integer|is|'
+                    r'|key|label|leap|less|lessOrEqual|literal|'
+                    r'|logical|lower|nand|negative|nor|not'
+                    r'|notEqual|null|numeric|odd|or|path'
+                    r'|pathLabel|positive|prefix|prime|set|some'
+                    r'|sorted|standalone|string|subset|suffix'
+                    r'|superset|ymbol|true|try|type|unless|upper'
+                    r'|when|whitespace|word|xnor|xor|zero'
+                    r')\b\?', Name.Builtin
+                ),
+            ],
+
+    }
+
+    def __init__(self, **options):
+        self.handle_annotateds = get_bool_opt(options, 'handle_annotateds', True)
+        RegexLexer.__init__(self, **options)
+
+__all__ = [
```

Comment:
Please put this near the top of the file, after the imports.

Reply:
[d80b451](https://github.com/pygments/pygments/pull/2259/commits/d80b451d702275f94707a12044feb5f22568e1de), done!